src/components/organisms/SettingsManager/Settings.tsx

```diff
@@ -75,6 +78,8 @@ export const Settings = ({
     Boolean(!config?.kubeConfig?.path) || Boolean(!config?.kubeConfig?.isPathValid)
   );
   const [currentKubeConfig, setCurrentKubeConfig] = useState(config?.kubeConfig?.path);
+  const [currentKubeConfigInValue, setCurrentKubeConfigInValue] = useState(config?.kubeConfig?.path);
+  const [spinBrowse, setspinBrowse] = useState(true);
```

Comment:
Lets rename this to `isKubeConfigPathChanging`

Reply:
Okay